### PR TITLE
Add git cheat-sheet in russian language

### PR DESCRIPTION
**Pull request by Tgjmjgj**
I found that the Git cheat sheet is here in many languages, but not in Russian.

 And I want to fix it!

But I have a question, how can I create its PDF version? There must be some kind of template or at least the necessary images to convert Markdown to PDF.

### CODE REVIEW

**Comment by brianamarie** [comment]
Hi @Tgjmjgj! Thank you for putting the work into this. I'm going to look into the PDF side of things, but either way, I don't want the PDF to hold up getting this merged at least as markdown. 

Generally, we like to have these reviewed by another native language speaker before we merge. Do you happen to know any other developers who speak Russian that would be willing to give this a review? 

cc @allthedoll for possible review in Russian 👀,  cc @hectorsector for knowledge about PDF

**Comment by Tgjmjgj** [comment]
Yea, I gave a look at this pair of people. Founded 2 typos, now I fixed it.

**Comment by brianamarie** [comment]
Thank you @Tgjmjgj. I'm going to reach out internally to a few people to see if I can get one more set of eyes, but if not, I'll get back to you soon so that we can get this merged in. 😄

File: downloads/ru/github-git-cheat-sheet.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+
+### Git для всех платформ
+http://git-scm.com
+
+## Первоначальная настройка
+Настройка информации о пользователе для всех локальных репозиториев
+
+```$ git config --global user.name "[имя]"```
+
+Устанавливает имя, которое будет отображаться в поле автора у выполняемых вами коммитов
```
Comment (rkzm) [inline downloads/ru/github-git-cheat-sheet.md]: Consider using less Деепричастие... A cheat sheet is usually simple...

Устанавливает имя, которое будет отображаться в поле автора, выполняемого коммиты

Comment (allthedoll) [reply downloads/ru/github-git-cheat-sheet.md]: This is actually comparative to the English cheat sheet, imho.

File: downloads/ru/github-git-cheat-sheet.md [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+Отменяет индексацию указанного файла, при этом сохраняет его содержимое
+
+
+```$ git commit -m "[сообщение с описанием]"```
+
+
+Фиксирует проиндексированные изменения и сохраняет их в историю версий
+
+## Коллективная работа
+Именованные серии коммитов и соединение результатов работы
```
Comment (rkzm) [inline downloads/ru/github-git-cheat-sheet.md]: список именованных веток комитов с указаниием ветки, на которой происходит работа в данный момент

File: downloads/ru/github-git-cheat-sheet.md [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+
+
+```$ git branch [имя ветки]```
+
+Создаёт новую ветку
+
+
+```$ git checkout [имя ветки]```
+
+Переключается на выбранную ветку и обновляет рабочую директорию до её состояния
```
Comment (rkzm) [inline downloads/ru/github-git-cheat-sheet.md]: Переключается на выбранную ветку с соответствующими коммитами

File: downloads/ru/github-git-cheat-sheet.md [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+
+
+```$ git merge [имя ветки]```
+
+Вливает изменения указанной ветки в текущую ветку
```
Comment (rkzm) [inline downloads/ru/github-git-cheat-sheet.md]: вносит instead of Вливает

File: downloads/ru/github-git-cheat-sheet.md [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+{% endcapture %}
+<div class="col-md-6">
+{{ colTwo | markdownify }}
+</div>
+<div class="clearfix"></div>
+
+
+{% capture colThree %}
+## Операции с файлами
+Перемещение и удаление версионированных файлов
```
Comment (rkzm) [inline downloads/ru/github-git-cheat-sheet.md]: Перемещение и удаление версий фаилов репозитория

File: downloads/ru/github-git-cheat-sheet.md [hunk 6]
```diff
@@ -0,0 +1,252 @@
...
+## Игнорирование некоторых файлов
+Исключение временных и вторичных файлов и директорий
+
+```
+*.log
+build/
+temp-*
+```
+
+Git будет игнорировать файлы и директории, перечисленные в файле `.gitignore` с примененим синтаксиса специальных паттернов
```
Comment (rkzm) [inline downloads/ru/github-git-cheat-sheet.md]: Git будет игнорировать файлы и директории, перечисленные в файле `.gitignore` с помощью wildcard

File: downloads/ru/github-git-cheat-sheet.md [hunk 7]
```diff
@@ -0,0 +1,252 @@
...
+
+
+```$ git stash pop```
+
+Восстанавливает состояние ранее сохранённых версий файлов
+
+
+```$ git stash list```
+
+Выводит список всех временных сохранений
```
Comment (rkzm) [inline downloads/ru/github-git-cheat-sheet.md]: временно сохраненных изменений

File: downloads/ru/github-git-cheat-sheet.md [hunk 8]
```diff
@@ -0,0 +1,252 @@
...
+
+
+```$ git stash drop```
+
+Сбрасывает последние сохраненные изменения
```
Comment (rkzm) [inline downloads/ru/github-git-cheat-sheet.md]: временно сохраненныe изменения

File: downloads/ru/github-git-cheat-sheet.md [hunk 9]
```diff
@@ -0,0 +1,252 @@
...
+
+
+```$ git log --follow [файл]```
+
+История изменений конкретного файла, включая его переименование
+
+
+```$ git diff [первая ветка]...[вторая ветка]```
+
+Показывает различия между двумя ветками
```
Comment (rkzm) [inline downloads/ru/github-git-cheat-sheet.md]: разница в содержании комитов во времени

File: downloads/ru/github-git-cheat-sheet.md (hunk 1 above)
Comment (rkzm) [reply downloads/ru/github-git-cheat-sheet.md]: This is production ready if you ask me :shipit:  ...
I have no Russian keyboard so finding it really hard typing. I would like to have a zoom session and knock this one out... 🤗 
Take my suggestions below with loads of salt 🤗 🤗

**Comment by brianamarie** [comment]
Wow! Great! I understand almost nothing on this page, but with the english I do understand, this looks good to ship. 🚢 Thank you so much @Tgjmjgj, and also @allthedoll and @rkzm for reviewing.

File: downloads/ru/github-git-cheat-sheet.md (hunk 1 above)
Comment (Tgjmjgj) [reply downloads/ru/github-git-cheat-sheet.md]: In most cases, your choice is really better. I use it, thanks!

Comment (Tgjmjgj) [reply downloads/ru/github-git-cheat-sheet.md]: If this closed, I need a new PR?

Comment (brianamarie) [reply downloads/ru/github-git-cheat-sheet.md]: Hello @Tgjmjgj! I was too excited to get this merged. Yes, to include new suggestions, we would need a new PR.

**Pull request merged**